edge-modules/edge-dashboard/src/state.rs

```diff
@@ -80,9 +80,9 @@ pub fn parse_query(query: &str, pair_delim: char, kv_delim: char) -> HashMap<&st
         .collect()
 }
 
-// returns the contents of the config.yaml file from the current device
+// returns the contents of the config.toml file from the current device
```

Comment:
Revert the changes to edge-dashboard. It doesn't even compile; no point making more diffs in it.

Reply:
fixed